src/raster/mdarray.rs

```diff
@@ -413,7 +424,7 @@ impl<'a> Group<'a> {
         }
     }
 
-    pub fn open_group(&self, name: &str, options: CslStringList) -> Result<Group> {
+    pub fn open_group(&'_ self, name: &str, options: CslStringList) -> Result<Group<'a>> {
```

Comment:
I'm a bit groggy right now, does `self` here have a different lifetime from the resulting `Group`?

Reply:
Yes, I prevented Group from having the lifetime of self. It needs the lifetime of the dataset.
I tested that in the changed test where I dropped a super-group.